x-pack/heartbeat/monitors/browser/synthexec/enrich.go

```diff
@@ -105,6 +105,16 @@ func (je *journeyEnricher) enrichSynthEvent(event *beat.Event, se *SynthEvent) e
 		}
 	}
 
+	if je.journey != nil {
+		eventext.MergeEventFields(event, mapstr.M{
+			"monitor": mapstr.M{
+				"check_group": je.checkGroup,
+				"id":          je.journey.ID,
+				"name":        je.journey.Name,
```

Comment:
need type as well. Otherwise the metrics information with logs would throw error - https://github.com/elastic/beats/blob/4959e3bb615e6788a7c943eae89019d947e1344b/heartbeat/monitors/logger/logger.go#L68
```
"message":"Error gathering information to log event","service.name":"heartbeat","logErrors":[{"error":"key not found"}]
```

Reply:
Fixed by moving the logRun in to wrappers.